x-pack/plugins/observability/server/plugin.ts

```diff
@@ -48,10 +49,16 @@ export class ObservabilityPlugin implements Plugin<ObservabilityPluginSetup> {
     }
 
     return {
-      getScopedAnnotationsClient: async (...args) => {
+      getScopedAnnotationsClient: async (...args: Parameters<ScopedAnnotationsClientFactory>) => {
         const api = await annotationsApiPromise;
         return api?.getScopedAnnotationsClient(...args);
       },
+      registry: plugins.ruleRegistry.create({
```

Comment:
`registry` isn't really clear. registry of what? Would `ruleRegistry` be ok or does that make it confusing because of the plugin with the same name?

Reply:
Yeah good point, will change!